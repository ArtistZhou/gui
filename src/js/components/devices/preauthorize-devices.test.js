import React from 'react';
import renderer from 'react-test-renderer';
import Preauthorize from './preauthorize-devices';

it('renders correctly', () => {
  const tree = renderer.create(<Preauthorize />).toJSON();
  expect(tree).toMatchSnapshot();
});
