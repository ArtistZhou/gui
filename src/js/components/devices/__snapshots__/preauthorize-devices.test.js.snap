// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="tab-container"
>
  <button
    className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-containedSecondary-14 MuiButton-raised-15 MuiButton-raisedSecondary-17 top-right-button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "position": "absolute",
      }
    }
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-2"
    >
      Preauthorize devices
    </span>
    <span
      className="MuiTouchRipple-root-46"
    />
  </button>
  <div
    className="loaderContainer"
  >
    <div
      className="loader"
    >
      <span
        className="dot dot_1"
      />
      <span
        className="dot dot_2"
      />
      <span
        className="dot dot_3"
      />
      <span
        className="dot dot_4"
      />
    </div>
  </div>
  <div
    className="hidden"
  >
    <p>
      There are no preauthorized devices.
    </p>
    <p>
      <a
        onClick={[Function]}
      >
        Preauthorize devices
      </a>
       so that when they come online, they will connect to the server immediately
    </p>
    <img
      alt="preauthorize"
      src="assets/img/preauthorize.png"
    />
  </div>
</div>
`;
